framework/TSA/Wavelet.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, *args, **kwargs):
       @ Out, None
     """
     # general infrastructure
-    TimeSeriesAnalyzer.__init__(self, *args, **kwargs)
+    super().__init__(self, *args, **kwargs)
```

Comment:
self in super call (sorry if I am wrong about this)

Reply:
I think you used to need a self argument in Python2 but in Python 3 that may no longer be needed. Although, I may be not understanding the post fully. Here is a SO citing the change between 2 and 3. 

https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods

Reply:
yeah, I tried this out:
```python
class A:
  def __init__(self, a, b):
    self._a = a
    self._b = b

class B(A):
  def __init__(self, a, b, c):
    super().__init__(self, a, b)
    self._c = c

b = B(1, 2, 3)
```
fails with `TypeError: __init__() takes 3 positional arguments but 4 were given`

Reply:
fixed